etc/compile/compile.sh

```diff
@@ -16,5 +17,13 @@ go build \
   -o _tmp/${BINARY} \
   -ldflags "${LD_FLAGS}" \
   src/server/cmd/${BINARY}/main.go
-docker-compose build ${BINARY}
-docker tag -f pachyderm_${BINARY}:latest pachyderm/${BINARY}:latest
+
+# When creating profile binaries, we dont want to detach or do docker ops
+if [ -z ${PROFILE} ]
+then
+    docker-compose build ${BINARY}
+    docker tag -f pachyderm_${BINARY}:latest pachyderm/${BINARY}:latest
+fi
+
+tar cf - _tmp/${BINARY}
```

Comment:
This should probably be in an else block right? Won't `make docker-build-pachd` emit a tar archive to stdout in this patch?

Also if you do 

```
cd _tmp
tar cf - ${BINARY}
```

then when we untar it we'll have just a binary and no directory, seems a bit nicer to me.